evennia/comms/models.py

```diff
@@ -646,8 +646,6 @@ def online(self):
             from django.core.exceptions import ObjectDoesNotExist
 
             try:
-                if hasattr(obj, "account") and obj.account:
-                    obj = obj.account
```

Comment:
This got lost?

Reply:
That was deliberate.
Both Accounts and Objects have an is_connected property.

Reply:
Fair enough. 
(by the way - you can add comments to your own PR lines - explaining your reasoning for the reviewer to avoid unnecessary questions).